jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -28,15 +28,21 @@ describe('Charmbrowser', function() {
 
   beforeAll(function(done) {
     // By loading this file it adds the component to the juju components.
-    YUI().use('charmbrowser-component', function() { done(); });
+    YUI().use([
+      'charmbrowser-component',
+      'entity-details',
+      'mid-point',
+      'search-results'
```

Comment:
Why are you including so many components here? Using the shallow renderer it won't render the sub components so they aren't necessary. See the inspector tests